chalicelib/checks/higlass_checks.py

```diff
@@ -271,39 +249,42 @@ def patch_files_for_higlass_viewconf(connection, **kwargs):
             if kwargs['file_accession'] and file_accession != kwargs['file_accession']:
                 continue
 
-            static_content_section = target_files_by_ga[ga][ file_accession ]
-
             # Post a new Higlass viewconf using the file list
             higlass_title = "{acc} - Higlass Viewconfig".format(acc=file_accession)
 
-            post_viewconf_results = post_viewconf_to_visualization_endpoint(connection, ref_files, [file_accession], ff_auth, headers, higlass_title)
+            post_viewconf_results = post_viewconf_to_visualization_endpoint(connection, ref_files, [file_accession,], ff_auth, headers, higlass_title)
 
             if post_viewconf_results["error"]:
                 action_logs['failed_to_create_viewconf'][file_accession] = post_viewconf_results["error"]
                 continue
 
             # Create a new static content section with the description = "auto_generated_higlass_view_config" and the new viewconf as the content
             # Patch the ExpSet static content
-            successful_patch, patch_error =  add_viewconf_static_content_to_file(connection, file_accession, post_viewconf_results["view_config_uuid"],
-            static_content_section,
-            "tab:higlass")
+            successful_patch, patch_error =  add_viewconf_static_content_to_file(
+                connection,
+                file_accession,
+                post_viewconf_results["view_config_uuid"],
+                static_content_section,
+                "tab:higlass"
+            )
 
             if not successful_patch:
                 action_logs['failed_to_patch_file'][file_accession] = patch_error
                 continue
 
             action_logs['new_view_confs_by_file'][file_accession] = post_viewconf_results["view_config_uuid"]
 
-    action.status = 'DONE'
-    action.output = action_logs
+    check.status = 'PASS'
+    check.full_output = action_logs
 
-    target_files_by_ga = gen_check_result['full_output'].get('target_files', {})
     file_count = sum([len(target_files_by_ga[ga]) for ga in target_files_by_ga])
-    action.progress = "Created Higlass viewconfs for {completed} out of {possible} files".format(
+    check.summary = "Created Higlass viewconfs for {completed} out of {possible} files".format(
         completed=len(action_logs["new_view_confs_by_file"].keys()),
         possible=file_count
     )
-    return action
+    check.description = check.summary
+    check.allow_action = False
```

Comment:
Not needed -- `check.allow_action` defaults to False; won't do anything if there isn't a check.action set anyways